src/benchmark_register.cc

```diff
@@ -162,8 +163,15 @@ bool BenchmarkFamilies::FindBenchmarks(
                   StringPrintF("%s:", family->arg_names_[arg_i].c_str());
             }
           }
-
+          
+#if defined(__ANDROID__) && defined(__GLIBCXX__)
+          // workaround for Android, http://stackoverflow.com/questions/22774009/android-ndk-stdto-string-support
+          std::ostringstream ss;
+          ss << arg;
+          instance.name += ss.str();
+#else
           instance.name += std::to_string(arg);
```

Comment:
How about `instance.name += FormatString("%d", arg)` instead?

I just want to avoid as much conditional compilation as possible. The more of it the library accumulates the harder it gets to maintain.